res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -312,6 +312,10 @@ limitations under the License.
     }
 }
 
+.mx_EventTile.mx_EventTile_noBubble[data-layout=bubble] {
+    --backgroundColor: tranparent;
```

Comment:
typo: missing "s"

Reply:
well that explains why it didn't syntax highlight, hilariously enough it works regardless :D

Reply:
```suggestion
    --backgroundColor: transparent;
```